app/models/cybersource/payment_response.rb

```diff
@@ -2,13 +2,14 @@
 
 module Cybersource
   # Response sent back by Cybersource after the external checkout is complete
+  # See the PDF linked in the README for more information on the fields
   class PaymentResponse
     class PaymentFailed < StandardError; end
 
     def initialize(fields)
       @fields = fields
       @signed_fields = fields['signed_field_names'].split(',')
-      @unsigned_fields = fields['unsigned_field_names'].split(',')
+      @unsigned_fields = fields['unsigned_field_names']&.split(',') || []
```

Comment:
Cybersource always signs _some_ fields, but may elect to sign _all_ of them, in which case `unsigned_field_names` is apparently not sent. TIL

Reply:
That would be a great thing to put in a comment in the code.

Reply:
Added.